packages/node_modules/@webex/plugin-meetings/src/meeting/muteState.js

```diff
@@ -173,8 +173,8 @@ class MuteState {
    * @returns {Promise}
    */
   sendLocalMuteRequestToServer(meeting) {
-    const audioMuted = (this.type === AUDIO) ? this.state.client.localMute : undefined;
-    const videoMuted = (this.type === VIDEO) ? this.state.client.localMute : undefined;
+    const audioMuted = (this.type === AUDIO) ? this.state.client.localMute : meeting.isAudioMuted();
```

Comment:
shouldn't we push back on this as a backend bug? when client does audio mute we set videoMuted to undefined, so the API request sent to locus /media endpoint only has the audioMuted property set. If as a result of that the video mute status in locus is changed that looks like incorrect Locus behaviour to me....

Reply:
also I think this can cause a problem: isAudioMuted() ends up calling MuteState.isMuted() and that one returns true also in the case if audio is only remotely muted but not locally muted. So in a scenario where the app asked to mute video and audio was remotely muted we would end up doing local mute for both audio and video. So we probably want to use here `meeting.audio?.state.client.localMute`.

Reply:
If we do `meeting.audio?.state.client.localMute` in the scenario you outlined above, we would unmute the remote audio. Is that what we want?

Reply:
as discussed offline, Locus has 2 separate APIs: one for local mute and one for remote mute, so we wouldn't affect the remote mute state